R/stat-quantile.r

```diff
@@ -54,7 +54,8 @@ StatQuantile <- ggproto("StatQuantile", Stat,
 
     if (is.null(formula)) {
       if (method == "rqss") {
-        try_require("MatrixModels", "stat_quantile")
+        # we need to attach quantreg for qss to work inside formula
+        require("quantreg")
```

Comment:
This still needs `try_require()` to get the nice error message

Reply:
No, we just called it before entering the outer `if` statement:
https://github.com/tidyverse/ggplot2/blob/8d02957ffed9816046cd85ff0ea2952b4e6a7ad7/R/stat-quantile.r#L53-L61

The `try_require()` for MatrixModels is not needed at all, as far as I can tell, because we are not directly calling any functions from MatrixModels and quantreg imports MatrixModels.